app/models/infra_conversion_job.rb

```diff
@@ -214,6 +229,14 @@ def order_ansible_playbook_service
     service_template.provision_request(migration_task.userid.to_i, service_dialog_options)
   end
 
+  def apply_right_sizing_cpu(mode)
+    destination_vm.set_number_of_cpus(source_vm.send("#{mode}_recommended_vcpus"))
+  end
+
+  def apply_right_sizing_memory(mode)
+    destination_vm.set_memory(source_vm.send("#{mode}_recommended_mem"))
+  end
+
```

Comment:
Aren't these already in https://github.com/ManageIQ/manageiq/pull/19238? Not a big deal, just thought it was odd.